VoiceController.php

```diff
@@ -1,48 +1,53 @@
-public function voice(Request $request){
-    $request->validate([
-        'question_id'=>'required|int|exists:questions,id',
-        'value'=>'required|boolean',
-    ]);
+<?php
+public function voice(VoiceRequest $request){
+    
+    // Moved requset to validation
+    $request->validated();
```

Comment:
@debjit good move, but then you don't need to call `->validated()` from the controller, validation happens automatically.